web/gds-user-ui/src/components/ChooseAnOrganization/index.tsx

```diff
@@ -76,6 +143,14 @@ function ChooseAnOrganization() {
           </Stack>
         </Stack>
       </Stack>
+      <HStack>
+        <Button onClick={PreviousPage} disabled={currentPage === 1}>
```

Comment:
We can create here something like `isCurrentPage` and a it inside a hooks

```suggestion
        <Button onClick={PreviousPage} disabled={isCurrentPage}>
```